### DISCUSSION TITLE
[Feature] Have "Replace Photo with upload" not update the meta data (specifically the Date)

### DISCUSSION THREAD

**Discussion by rkirmeier** [immich-app/immich#14356] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the current state of the software, there is no web option to edit photos, so in the mean time, I am relying on the "replace with upload" option to do any edits. the main edit I am doing is going through and changing orientation. I download the photo, do my edit, then use "replace with upload". This task is doubled by this feature overwriting the meta data. i have to snip it to the side and reinput it. I think this is only when the data was originally pulled from the photo (which is probably most of the time for me). 

I think that if you are using the replace photo feature, it would make sense that the meta data doesn't need to be pulled from the new photo (that inevitably says it was created the day I edited it). Correct me if I am wrong, but i think that is the intended use for such a feature. right now, I might as well be just moving the old version to the trash and uploading a new copy as it stands.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mertalev** [comment]
It's the only sane way to handle the image content being different. Features should be able to rely on asset metadata being accurate and not left behind from the previous asset. Maybe editable attributes like date and location could be handled differently.

**Reply by rkirmeier** [reply]
I see that point of view. i think someone might be able to approach this differently than I but I think if someone wanted the new metadata, they would just upload it as a new image. the only reason to use a feature like this would be if you wanted to keep some information about the picture prior to its re upload. If not by default or setting, I think it should at least ask you what metadata to keep and what to replace.

**Reply by mertalev** [reply]
I think this feature was mainly intended for preserving album membership, tags, etc. and not really about metadata.

**Reply by rkirmeier** [reply]
i would consider tags metadata, even if it is not something part of the original file. Also, facial recognition data stays (to some detriment). I think there is a balance here that can be struck

**Comment by helgehatt** [comment]
I have used this feature specifically to update the metadata of existing photos in Immich. Source photos were missing date, so I changed the dates using exiftool and uploaded with replace.

I think you should rather find a way to edit your photos while preserving metadata.

**Comment by bo0tzz** [comment]
This endpoint has been deprecated and will eventually be removed, so I'm closing this request.

**Reply by rkirmeier** [reply]
Apologies, I don't understand the lingo. Is a feature being depricated?

**Comment by murray484** [comment]
I just ran into this.  Is there a new way we should be replacing photos if we want to keep the metadata?  My use case is scanning old photos for archival.  When we scan the image we enter all the metadata manually (usually it's written on the back of the original photo).  Later on, I will download the scanned photo, color correct it in photoshop, and use the replace with upload feature to put it back. But this wipes out the metadata.  Obviously this is less than ideal...
